### PR TITLE
NC news 5

### PR DESCRIPTION
**Pull request by tr-tobi**
I was struggling to figure out what the errors could be for this path

### CODE REVIEW

**Comment by Harrison365** [comment]
Hi Tobi :) 

This looks great 🎉 

However, there are a couple of things to fix:
-Please get and format all of the necessary data using your SQL query. I.e. don't delete things using JavaScript after the query has returned data. 
-Please add more tests for the keys that come back. We want to be sure that the articles have all of the expected keys before we merge. 

Please send another PR when you think you have done these. 

Thanks 👍

File: __tests__/app.test.js [hunk 1]
```diff
@@ -36,6 +36,55 @@ describe("/topic", () => {
...
+        expect(res.body.articles[5]).toEqual({
+          author: "rogersop",
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          article_id: 5,
+          topic: "cats",
+          created_at: "2020-08-03T13:14:00.000Z",
+          votes: 0,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "2",
```
Comment (johnhutcheon) [inline __tests__/app.test.js]: We don't need lines 61 to 82 in this test. What you've done above this is fine and shows that the endpoint works! 🙂

File: __tests__/app.test.js [hunk 2]
```diff
@@ -36,6 +36,55 @@ describe("/topic", () => {
...
 
+describe("/articles", () => {
+  test("GET:200 sends an array of the article objects to the end point ", () => {
+    return req(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        console.log(res.body);
+        expect(res.body.articles).toBeSortedBy("created_at", {
+          descending: true,
```
Comment (johnhutcheon) [inline __tests__/app.test.js:47]: Nice work 👍

**Review by johnhutcheon — APPROVED** [review]
Great work Tobi! Just delete the part of the test i've mentioned in my comment, then happy for you to merge! 👍

**Pull request merged**